mcpele/monte_carlo/_action_cpp.pyx

```diff
@@ -201,6 +229,11 @@ class RecordPairDistHistogram(_Cdef_RecordPairDistHistogram):
         number of equilibration steps to be excluded from :math:`g(r)` computation
     record_every : int
         after ``eqsteps`` steps have been done, record every ``record_everyth`` steps
+    opt : pele graident optimizer (optimal)
```

Comment:
i would call it optimizer rather than opt.  opt can be confused with too many things.  e.g. options